html/components/accordion.stories.js

```diff
@@ -33,8 +33,7 @@ export const defaultStory = () => {
   return `
 <ul class="sprk-c-Accordion sprk-o-VerticalList">
   <li class="sprk-c-Accordion__item" data-sprk-toggle="container">
-    <a
-      aria-controls="details-one"
+    <button
```

Comment:
Hey @bobwise , why did we remove the aria-controls?

Reply:
aria controls is autogenerated by generateAriaControls now

Reply:
I see you note now in the PR. We should leave those on there. In case JS is off?

Reply:
We need JS to do the controlling. If we turn JS off, aria-controls should not be on the element.